include/grpc++/server.h

```diff
@@ -171,34 +190,31 @@ class Server GRPC_FINAL : public ServerInterface, private GrpcLibraryCodegen {
 
   const int max_receive_message_size_;
 
-  // Completion queue.
-  CompletionQueue cq_;
+  /// The following completion queues are ONLY used in case of Sync API i.e if
+  /// the server has any services with sync methods. The server uses these
+  /// completion queues to poll for new RPCs
+  std::shared_ptr<std::vector<std::unique_ptr<ServerCompletionQueue>>>
+      sync_server_cqs_;
+
+  /// List of ThreadManager instances (one for each cq in the sync_server_cqs)
+  std::vector<std::unique_ptr<SyncRequestThreadManager>> sync_req_mgrs_;
```

Comment:
No plan at this point. Might do this in future (once I find more use cases for ThreadManager interface and once that evolves a bit more)